example_pipeline/data_io.py

```diff
@@ -361,6 +366,8 @@ def _export_edf_hypnogram(file_path, states, intervals):
     with EdfWriter(file_path, 0) as writer:
         for (start, stop), state in zip(intervals, states):
             writer.writeAnnotation(start, stop-start, state)
+        if header:
+            writer.setHeader(header)
```

Comment:
Support for `header` keyword argument added in d0c0e1a6057812dfe2c861d06b45b3e8d194f2bb.